### DISCUSSION TITLE
We're still processing your application. Once this is complete you will have access to the student developer pack offers.

### DISCUSSION THREAD

**Discussion by naitik0009** [community/community#83791] [GitHub Education] [answered]
### Hi everyone,

We're still processing your application. Once this is complete you will have access to the student developer pack offers.
this is the notification i'm getting from 3-4 months and unable to reverify and apply for student developer pack when i try to reverify it opens pricing page please help

**Comment by criskkky** [comment]
Mmmh you should provide your detailed step by step, perhaps there is something that you are doing wrong or skipping

**Comment by samus-aran** [comment, marked as answer]
Hi @naitik0009 

We are seeing a higher volume of student developer pack inquiries coming through. We are working our way through addressing these as fast as possible. I can share that your request has been passed onto the GitHub Education team and will be processed as soon as possible, but I am afraid I do not have a timeline to share for when you can expect this to happen.

Usually with these types of tickets we may include an investigation as fraudulent activity has been identified on either the account or the associated school's. We thank you for your patience and understanding as we do our due diligence ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

**Reply by naitik0009** [reply]
Ok it's already the fourth week let's see how long will it take.

**Reply by thecoderkid351** [reply]
were you able to get access eventually? i am struggling as well

**Reply by ghost** [reply]
how long did it take
to start

**Reply by Deliciousp999** [reply]
> Ok it's already the fourth week let's see how long will it take. Do -ping enter class sinus 1 meter cosinus 30*celsius

**Reply by Deliciousp999** [reply]
Command for howlong

**Reply by Deliciousp999** [reply]
> Command for howlong

I hope you like thé additional
![1741319174754](https://github.com/user-attachments/assets/8cf904c5-1071-48b0-85c0-961aa0475e3c)
I think a timewatch predict sxx today

**Comment by Tender-sun** [comment]
Have you solved it yet?

**Reply by thecoderkid351** [reply]
sadly no

**Reply by Tender-sun** [reply]
how long this status stay?

**Reply by Deliciousp999** [reply]
I don't know till it Gert compromised 
![1741278698209](https://github.com/user-attachments/assets/a2b140eb-6d2c-4037-bfc8-7940467893df)

**Comment by Ialiekwu** [comment]
I've been in the same boat..........

**Reply by alisajad001** [reply]
Same here.

**Comment by anurag596** [comment]
I GOT IT!!!!
Try clicking photo of your ID on the verification page in the website only (don't select ID image from your storage.....i think it isn't working)

**Reply by alisajad001** [reply]
Yes successfuly I got it too.

**Reply by alasdair157** [reply]
Worked for me too! Thank you!

**Reply by rafoo123** [reply]
how?

**Reply by anurag596** [reply]
> how?
> 

Just click the photo of your id within the website then it will work ;)

**Reply by rafoo123** [reply]
nothing :(

**Reply by anurag596** [reply]
> nothing :(
> 

It takes time (around 4-5 days)

**Comment by dpaulos6** [comment]
If any of you guys still have trouble with the submission of your application, try taking a photo of your document instead of uploading.
For me uploading wasn't moving to the submission part, then i used DroidCam to use my phone as webcam, and took a photo of my document and it started loading the submission phase.

Also make sure you have 2FA enabled, and have your billing information setup. If the submission will fail and ask you to fix those before you can submit your application.

If this still didn't work, consider checking [this](https://github.com/orgs/community/discussions/111352) to find the reason why.

**Comment by alisajad001** [comment]
Guys when you take your ID photo you should wait for 3-4 days and it'll be approved. The key is to take your ID photo not to upload it.

**Comment by Chrysthy** [comment]
I'm struggling it too. I don't know what else I can do. 😢

"We're still processing your application. Once this is complete you will have access to the student developer pack offers."

**Reply by dpaulos6** [reply]
when did you make your application?

**Reply by Chrysthy** [reply]
It was on 08/20/2024, and I also received this email congratulating me, but it still says it's pending.

![github](https://github.com/user-attachments/assets/bed0a831-da1e-471a-bd9f-746ba4fe139f)

**Reply by dpaulos6** [reply]
have you tried accessing the links in the email see if anything shows up?

**Reply by alisajad001** [reply]
Please be patient!

**Reply by Chrysthy** [reply]
> have you tried accessing the links in the email see if anything shows up?

Yes, and it takes me to this page, and when I click on "Join," it leads me here. Do you know if I need to create an organization?

![photo-1](https://github.com/user-attachments/assets/8b046feb-b39b-4a87-ba34-26bdc7c90c24)
![photo-2](https://github.com/user-attachments/assets/6eee16c7-6cd1-415b-98d6-5803577736e0)

**Reply by Chrysthy** [reply]
> Please be patient!

I'm being patient, I'm not angry or upset. I just want to know if there's anything else I need to do. Have a good day!

**Reply by dpaulos6** [reply]
its been over a month and technically you received an email saying you were approved. send gh an email to confirm its not a bug or whatever. shouldn't take this long.

**Reply by Chrysthy** [reply]
> its been over a month and technically you received an email saying you were approved. send gh an email to confirm its not a bug or whatever. shouldn't take this long.

Thank you so much for you help! I'll do that. 😊

**Reply by alisajad001** [reply]
> > Please be patient!
> 
> I'm being patient, I'm not angry or upset. I just want to know if there's anything else I need to do. Have a good day!

Oh, sorry I thought it's 09/20/2024.

**Reply by TonyEinstein** [reply]
Hello, can you access the student package right now? I have the same problem as you. I have been waiting for the seventh day now. How long did it take you to access the permissions?

**Comment by yifan0414** [comment]
I got it too....
How to solve it?

**Reply by dpaulos6** [reply]
There's no way to fix it besides waiting. How long have you completed your application?

**Reply by anurag596** [reply]
https://github.com/community/community/discussions/83791#discussioncomment-9028742

Try this one

**Reply by yifan0414** [reply]
> [#83791 (comment)](https://github.com/orgs/community/discussions/83791#discussioncomment-9028742)
> 
> Try this one

I can not enter that website, when i click join eduction, it jump to the price web..

**Reply by TonyEinstein** [reply]
Hello, can you access the student package right now? I have the same problem as you. I have been waiting for the seventh day now. How long did it take you to access the permissions?

**Reply by dpaulos6** [reply]
> Hello, can you access the student package right now? I have the same problem as you. I have been waiting for the seventh day now. How long did it take you to access the permissions?

Have you received an email saying you were accepted? If not, can you make sure you actually submitted an application? If you're received an email saying you were approved, please wait until you can access it, there must be some sort of delay.

If this persists another week consider contacting GitHub education support or general support related to your situation, as it might be a bug if it takes too long.
